<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>A: Data resources and values for single cell genomics • SCecosys</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="A: Data resources and values for single cell genomics">
<meta property="og:description" content="SCecosys">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">SCecosys</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.0.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/SCecosys.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/A_Data.html">A: Data resources and values for single cell genomics</a>
    </li>
    <li>
      <a href="../articles/B_Analysis.html">B: Analysis -- OSCA resources, etc.</a>
    </li>
    <li>
      <a href="../articles/C_teaching.html">C: Teaching and learning single cell genomics</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/vjcitn/SCecosys/">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="A_Data_files/htmlwidgets-1.6.2/htmlwidgets.js"></script><link href="A_Data_files/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="A_Data_files/datatables-binding-0.27/datatables.js"></script><link href="A_Data_files/dt-core-1.12.1/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="A_Data_files/dt-core-1.12.1/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="A_Data_files/dt-core-1.12.1/js/jquery.dataTables.min.js"></script><link href="A_Data_files/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet">
<script src="A_Data_files/crosstalk-1.2.0/js/crosstalk.min.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>A: Data resources and values for single cell genomics</h1>
                        <h4 data-toc-skip class="author">Vincent J. Carey, stvjc at channing.harvard.edu</h4>
            
            <h4 data-toc-skip class="date">March 27, 2023</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/vjcitn/SCecosys/blob/HEAD/vignettes/A_Data.Rmd"><code>vignettes/A_Data.Rmd</code></a></small>
      <div class="hidden name"><code>A_Data.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="the-singlecellexperiment-class">The SingleCellExperiment class<a class="anchor" aria-label="anchor" href="#the-singlecellexperiment-class"></a>
</h2>
<div class="figure">
<img src="sceSchema.jpg" alt="sce"><p class="caption">sce</p>
</div>
</div>
<div class="section level2">
<h2 id="bioconductors-scrnaseq-resource">Bioconductor’s scRNAseq resource<a class="anchor" aria-label="anchor" href="#bioconductors-scrnaseq-resource"></a>
</h2>
<p>The scRNAseq package includes functions that will retrieve exemplary datasets, place them in a user-defined cache (so that retrievals need not be repeated), and deliver a representation into the workspace.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/vjcitn/SCecosys">SCecosys</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">scRNAseq</span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/rstudio/DT" class="external-link">DT</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/stemangiola/CuratedAtlasQueryR" class="external-link">CuratedAtlasQueryR</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://dplyr.tidyverse.org" class="external-link">dplyr</a></span><span class="op">)</span></span></code></pre></div>
<div class="section level3">
<h3 id="enumerating-the-functions-and-their-sources-in-pubmed">Enumerating the functions and their sources in pubmed<a class="anchor" aria-label="anchor" href="#enumerating-the-functions-and-their-sources-in-pubmed"></a>
</h3>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">curated_scrnaseq_table</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span><span class="va">curated_scrnaseq_table</span>, escape<span class="op">=</span><span class="cn">FALSE</span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-143f3fd9c8fdd16c5bf8" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-143f3fd9c8fdd16c5bf8">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61"],["AztekinTailData()","BachMammaryData()","BacherTCellData()","BaronPancreasData('human')","BaronPancreasData('mouse')","BhaduriOrganoidData()","BuettnerESCData()","BunisHSPCData()","CampbellBrainData()","ChenBrainData()","DarmanisBrainData()","ErnstSpermatogenesisData()","FletcherOlfactoryData()","GrunHSCData()","GrunPancreasData()","GiladiHSCData(mode='rna')","HeOrganAtlasData()","HermannSpermatogenesisData()","HuCortexData()","KolodziejczykESCData()","JessaBrainData()","LaMannoBrainData('human-es')","LaMannoBrainData('human-embryo')","LaMannoBrainData('human-ips')","LaMannoBrainData('mouse-adult')","LaMannoBrainData('mouse-embryo')","LawlorPancreasData()","LedergorMyelomaData()","LengESCData()","LunSpikeInData('416b')","LunSpikeInData('tropho')","MacoskoRetinaData()","ReprocessedTh2Data()","MairPBMCData()","KotliarovPBMCData()","MarquesBrainData()","MessmerESCData()","MuraroPancreasData()","NestorowaHSCData()","NowakowskiCortexData()","PaulHSCData()","ReprocessedFluidigmData()","PollenGliaData()","RichardTCellData()","RomanovBrainData()","SegerstolpePancreasData()","ShekharRetinaData()","StoeckiusHashingData(mode='mouse')","StoeckiusHashingData(mode='human')","StoeckiusHashingData(type='mixed')","UsoskinBrainData()","TasicBrainData()","ReprocessedAllenData()","WuKidneyData()","XinPancreasData()","ZeiselBrainData()","ZeiselNervousData()","ZhaoImmuneLiverData()","ZhongPrefrontalData()","ZilionisLungData()","ZilionisLungData('mouse')"],[8355,10090,9606,9606,10090,9606,10090,9606,10090,10090,9606,10090,10090,10090,9606,10090,9606,10090,10090,10090,10090,9606,9606,9606,10090,9606,9606,9606,9606,10090,10090,10090,10090,9606,9606,10090,9606,9606,10090,9606,10090,9606,9606,10090,10090,9606,10090,10090,9606,9606,10090,10090,10090,10090,9606,10090,10090,9606,9606,9606,10090],["tail","mammary gland","T cells","pancreas","pancreas","cortical organoids","embryonic stem cells","haematopoietic stem and progenitor","brain","brain","brain","testis","olfactory epithelium","haematopoietic stem cells","pancreas","haematopoietic stem cells","various organs","spermatogenic cells","cortex","embryonic stem cells","brain","embryonic stem cells","embryonic midbrain","induced pluripotent stem cells","adult dopaminergic neurons","embyronic midbrain","pancreas","bone marrow plasma cells","embryonic stem cells","416B cells","trophoblasts","retina","T helper cells","peripheral blood mononuclear cells","peripheral blood mononuclear cells","brain","embryonic stem cells","pancreas","haematopoietic stem cells","cortex","haematopoietic stem cells","cortex","outer radial glia","CD8+ T cells","brain","pancreas","retina","peripheral blood mononuclear cells","peripheral blood mononuclear cells","HEK, THP1, K562, KG1 cells","brain","brain","visual cortex","kidney","pancreas","brain","nervous system","liver immune cells","prefrontal cortex","lung","lung"],[13199,25806,104417,8569,1886,242349,288,5183,21086,14437,466,68937,616,1915,1728,81024,84363,2325,48000,704,61595,1715,1977,337,243,1907,638,51840,460,192,192,49300,96,29033,58654,5069,1344,3072,1920,4261,10368,65,367,572,2881,3514,44994,50000,50000,30000,864,1809,379,17542,1600,3005,160796,68100,2394,173954,17549],["<a href=\"https://pubmed.ncbi.nlm.nih.gov/31097661\">31097661<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/29225342\">29225342<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/33296686\">33296686<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27667365\">27667365<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27667365\">27667365<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/31996853\">31996853<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/25599176\">25599176<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/33406429\">33406429<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/28166221\">28166221<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/28355573\">28355573<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/26060301\">26060301<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30890697\">30890697<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/28506465\">28506465<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27345837\">27345837<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27345837\">27345837<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/29915358\">29915358<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/33287869\">33287869<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30404016\">30404016<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/29220646\">29220646<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/26431182\">26431182<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/31768071\">31768071<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27716510\">27716510<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27716510\">27716510<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27716510\">27716510<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27716510\">27716510<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27716510\">27716510<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27864352\">27864352<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30523328\">30523328<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/26301841\">26301841<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/29030468\">29030468<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/29030468\">29030468<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/26000488\">26000488<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/24813893\">24813893<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/32268080\">32268080<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/32094927\">32094927<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27284195\">27284195<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30673604\">30673604<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27693023\">27693023<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27365425\">27365425<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/29217575\">29217575<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/26627738\">26627738<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/25086649\">25086649<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/26406371\">26406371<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30013148\">30013148<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27991900\">27991900<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27667667\">27667667<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27565351\">27565351<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30567574\">30567574<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30567574\">30567574<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30567574\">30567574<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/25420068\">25420068<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/26727548\">26727548<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/26727548\">26727548<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30510133\">30510133<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27667665\">27667665<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/25700174\">25700174<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30096314\">30096314<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/32351704\">32351704<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/29539641\">29539641<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30979687\">30979687<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30979687\">30979687<\/a>"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Call<\/th>\n      <th>Taxonomy<\/th>\n      <th>Part<\/th>\n      <th>Number<\/th>\n      <th>pmid<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="section level3">
<h3 id="searchable-abstracts-specially-curated">Searchable abstracts (specially curated)<a class="anchor" aria-label="anchor" href="#searchable-abstracts-specially-curated"></a>
</h3>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">curated_abstdf</span><span class="op">)</span></span>
<span><span class="va">curated_abstdf</span><span class="op">$</span><span class="va">pmids</span> <span class="op">=</span> <span class="fu"><a href="../reference/build_anchors.html">build_anchors</a></span><span class="op">(</span><span class="va">curated_abstdf</span><span class="op">$</span><span class="va">pmids</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span><span class="va">curated_abstdf</span>, escape<span class="op">=</span><span class="cn">FALSE</span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-da61e8d5a84505e11dab" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-da61e8d5a84505e11dab">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50"],["<a href=\"https://pubmed.ncbi.nlm.nih.gov/31097661\">31097661<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/29225342\">29225342<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/33296686\">33296686<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27667365\">27667365<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/31996853\">31996853<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/25599176\">25599176<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/33406429\">33406429<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/28166221\">28166221<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/28355573\">28355573<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/26060301\">26060301<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30890697\">30890697<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/28506465\">28506465<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27345837\">27345837<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/29915358\">29915358<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/33287869\">33287869<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30404016\">30404016<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/29220646\">29220646<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/26431182\">26431182<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/31768071\">31768071<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27716510\">27716510<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27864352\">27864352<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30523328\">30523328<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/26301841\">26301841<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/29030468\">29030468<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/26000488\">26000488<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/24813893\">24813893<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/32268080\">32268080<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/32094927\">32094927<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27284195\">27284195<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30673604\">30673604<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27693023\">27693023<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27365425\">27365425<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/29217575\">29217575<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/26627738\">26627738<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/25086649\">25086649<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/26406371\">26406371<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30013148\">30013148<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27991900\">27991900<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27667667\">27667667<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27565351\">27565351<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30567574\">30567574<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/25420068\">25420068<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/26727548\">26727548<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30510133\">30510133<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/27667665\">27667665<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/25700174\">25700174<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30096314\">30096314<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/32351704\">32351704<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/29539641\">29539641<\/a>","<a href=\"https://pubmed.ncbi.nlm.nih.gov/30979687\">30979687<\/a>"],["Unlike mammals, Xenopus laevis tadpoles have a high regenerative potential. To characterize this regenerative response, we performed single-cell RNA sequencing after tail amputation. By comparing naturally occurring regeneration-competent and -incompetent tadpoles, we identified a previously unrecognized cell type, which we term the regeneration-organizing cell (ROC). ROCs are present in the epidermis during normal tail development and specifically relocalize to the amputation plane of regeneration-competent tadpoles, forming the wound epidermis. Genetic ablation or manual removal of ROCs blocks regeneration, whereas transplantation of ROC-containing grafts induces ectopic outgrowths in early embryos. Transcriptional profiling revealed that ROCs secrete ligands associated with key regenerative pathways, signaling to progenitors to reconstitute lost tissue. These findings reveal the cellular mechanism through which ROCs form the wound epidermis and ensure successful regeneration.","Characterising the hierarchy of mammary epithelial cells (MECs) and how they are regulated during adult development is important for understanding how breast cancer arises. Here we report the use of single-cell RNA sequencing to determine the gene expression profile of MECs across four developmental stages; nulliparous, mid gestation, lactation and post involution. Our analysis of 23,184 cells identifies 15 clusters, few of which could be fully characterised by a single marker gene. We argue instead that the epithelial cells-especially in the luminal compartment-should rather be conceptualised as being part of a continuous spectrum of differentiation. Furthermore, our data support the existence of a common luminal progenitor cell giving rise to intermediate, restricted alveolar and hormone-sensing progenitors. This luminal progenitor compartment undergoes transcriptional changes in response to a full pregnancy, lactation and involution. In summary, our results provide a global, unbiased view of adult mammary gland development.","CD4+ T cells reactive against SARS-CoV-2 can be found in unexposed individuals, and these are suggested to arise in response to common cold coronavirus (CCCoV) infection. Here, we utilized SARS-CoV-2-reactive CD4+ T cell enrichment to examine the antigen avidity and clonality of these cells, as well as the relative contribution of CCCoV cross-reactivity. SARS-CoV-2-reactive CD4+ memory T cells were present in virtually all unexposed individuals examined, displaying low functional avidity and multiple, highly variable cross-reactivities that were not restricted to CCCoVs. SARS-CoV-2-reactive CD4+ T cells from COVID-19 patients lacked cross-reactivity to CCCoVs, irrespective of strong memory T cell responses against CCCoV in all donors analyzed. In severe but not mild COVID-19, SARS-CoV-2-specific T cells displayed low functional avidity and clonality, despite increased frequencies. Our findings identify low-avidity CD4+ T cell responses as a hallmark of severe COVID-19 and argue against a protective role for CCCoV-reactive T cells in SARS-CoV-2 infection.","Although the function of the mammalian pancreas hinges on complex interactions of distinct cell types, gene expression profiles have primarily been described with bulk mixtures. Here we implemented a droplet-based, single-cell RNA-seq method to determine the transcriptomes of over 12,000 individual pancreatic cells from four human donors and two mouse strains. Cells could be divided into 15 clusters that matched previously characterized cell types: all endocrine cell types, including rare epsilon-cells; exocrine cell types; vascular cells; Schwann cells; quiescent and activated stellate cells; and four types of immune cells. We detected subpopulations of ductal cells with distinct expression profiles and validated their existence with immuno-histochemistry stains. Moreover, among human beta- cells, we detected heterogeneity in the regulation of genes relating to functional maturation and levels of ER stress. Finally, we deconvolved bulk gene expression samples using the single-cell data to detect disease-associated differential expression. Our dataset provides a resource for the discovery of novel cell type-specific transcription factors, signaling receptors, and medically relevant genes.","Cortical organoids are self-organizing three-dimensional cultures that model features of the developing human cerebral cortex1,2. However, the fidelity of organoid models remains unclear3-5. Here we analyse the transcriptomes of individual primary human cortical cells from different developmental periods and cortical areas. We find that cortical development is characterized by progenitor maturation trajectories, the emergence of diverse cell subtypes and areal specification of newborn neurons. By contrast, organoids contain broad cell classes, but do not recapitulate distinct cellular subtype identities and appropriate progenitor maturation. Although the molecular signatures of cortical areas emerge in organoid neurons, they are not spatially segregated. Organoids also ectopically activate cellular stress pathways, which impairs cell-type specification. However, organoid stress and subtype defects are alleviated by transplantation into the mouse cortex. Together, these datasets and analytical tools provide a framework for evaluating and improving the accuracy of cortical organoids as models of human brain development.","Recent technical developments have enabled the transcriptomes of hundreds of cells to be assayed in an unbiased manner, opening up the possibility that new subpopulations of cells can be found. However, the effects of potential confounding factors, such as the cell cycle, on the heterogeneity of gene expression and therefore on the ability to robustly identify subpopulations remain unclear. We present and validate a computational approach that uses latent variable models to account for such hidden factors. We show that our single-cell latent variable model (scLVM) allows the identification of otherwise undetectable subpopulations of cells that correspond to different stages during the differentiation of naive T cells into T helper 2 cells. Our approach can be used not only to identify cellular subpopulations but also to tease apart different sources of gene expression heterogeneity in single-cell transcriptomes.","Whereas the human fetal immune system is poised to generate immune tolerance and suppress inflammation in utero, an adult-like immune system emerges to orchestrate anti-pathogen immune responses in post-natal life. It has been posited that cells of the adult immune system arise as a discrete ontological \"layer\" of hematopoietic stem-progenitor cells (HSPCs) and their progeny; evidence supporting this model in humans has, however, been inconclusive. Here, we combine bulk and single-cell transcriptional profiling of lymphoid cells, myeloid cells, and HSPCs from fetal, perinatal, and adult developmental stages to demonstrate that the fetal-to-adult transition occurs progressively along a continuum of maturity-with a substantial degree of inter-individual variation at the time of birth-rather than via a transition between discrete waves. These findings have important implications for the design of strategies for prophylaxis against infection in the newborn and for the use of umbilical cord blood (UCB) in the setting of transplantation.","The hypothalamic arcuate-median eminence complex (Arc-ME) controls energy balance, fertility and growth through molecularly distinct cell types, many of which remain unknown. To catalog cell types in an unbiased way, we profiled gene expression in 20,921 individual cells in and around the adult mouse Arc-ME using Drop-seq. We identify 50 transcriptionally distinct Arc-ME cell populations, including a rare tanycyte population at the Arc-ME diffusion barrier, a new leptin-sensing neuron population, multiple agouti-related peptide (AgRP) and pro-opiomelanocortin (POMC) subtypes, and an orexigenic somatostatin neuron population. We extended Drop-seq to detect dynamic expression changes across relevant physiological perturbations, revealing cell type-specific responses to energy status, including distinct responses in AgRP and POMC neuron subtypes. Finally, integrating our data with human genome-wide association study data implicates two previously unknown neuron populations in the genetic control of obesity. This resource will accelerate biological discovery by providing insights into molecular and cell type diversity from which function can be inferred.","The hypothalamus is one of the most complex brain structures involved in homeostatic regulation. Defining cell composition and identifying cell-type-specific transcriptional features of the hypothalamus is essential for understanding its functions and related disorders. Here, we report single-cell RNA sequencing results of adult mouse hypothalamus, which defines 11 non-neuronal and 34 neuronal cell clusters with distinct transcriptional signatures. Analyses of cell-type-specific transcriptomes reveal gene expression dynamics underlying oligodendrocyte differentiation and tanycyte subtypes. Additionally, data analysis provides a comprehensive view of neuropeptide expression across hypothalamic neuronal subtypes and uncover Crabp1","The human brain is a tissue of vast complexity in terms of the cell types it comprises. Conventional approaches to classifying cell types in the human brain at single cell resolution have been limited to exploring relatively few markers and therefore have provided a limited molecular characterization of any given cell type. We used single cell RNA sequencing on 466 cells to capture the cellular complexity of the adult and fetal human brain at a whole transcriptome level. Healthy adult temporal lobe tissue was obtained during surgical procedures where otherwise normal tissue was removed to gain access to deeper hippocampal pathology in patients with medical refractory seizures. We were able to classify individual cells into all of the major neuronal, glial, and vascular cell types in the brain. We were able to divide neurons into individual communities and show that these communities preserve the categorization of interneuron subtypes that is typically observed with the use of classic interneuron markers. We then used single cell RNA sequencing on fetal human cortical neurons to identify genes that are differentially expressed between fetal and adult neurons and those genes that display an expression gradient that reflects the transition between replicating and quiescent fetal neuronal populations. Finally, we observed the expression of major histocompatibility complex type I genes in a subset of adult neurons, but not fetal neurons. The work presented here demonstrates the applicability of single cell RNA sequencing on the study of the adult human brain and constitutes a first step toward a comprehensive cellular atlas of the human brain.","Male gametes are generated through a specialised differentiation pathway involving a series of developmental transitions that are poorly characterised at the molecular level. Here, we use droplet-based single-cell RNA-Sequencing to profile spermatogenesis in adult animals and at multiple stages during juvenile development. By exploiting the first wave of spermatogenesis, we both precisely stage germ cell development and enrich for rare somatic cell-types and spermatogonia. To capture the full complexity of spermatogenesis including cells that have low transcriptional activity, we apply a statistical tool that identifies previously uncharacterised populations of leptotene and zygotene spermatocytes. Focusing on post-meiotic events, we characterise the temporal dynamics of X chromosome re-activation and profile the associated chromatin state using CUT&RUN. This identifies a set of genes strongly repressed by H3K9me3 in spermatocytes, which then undergo extensive chromatin remodelling post-meiosis, thus acquiring an active chromatin state and spermatid-specific expression.","A detailed understanding of the paths that stem cells traverse to generate mature progeny is vital for elucidating the mechanisms governing cell fate decisions and tissue homeostasis. Adult stem cells maintain and regenerate multiple mature cell lineages in the olfactory epithelium. Here we integrate single-cell RNA sequencing and robust statistical analyses with in vivo lineage tracing to define a detailed map of the postnatal olfactory epithelium, revealing cell fate potentials and branchpoints in olfactory stem cell lineage trajectories. Olfactory stem cells produce support cells via direct fate conversion in the absence of cell division, and their multipotency at the population level reflects collective unipotent cell fate decisions by single stem cells. We further demonstrate that Wnt signaling regulates stem cell fate by promoting neuronal fate choices. This integrated approach reveals the mechanisms guiding olfactory lineage trajectories and provides a model for deconstructing similar hierarchies in other stem cell niches.","Adult mitotic tissues like the intestine, skin, and blood undergo constant turnover throughout the life of an organism. Knowing the identity of the stem cell is crucial to understanding tissue homeostasis and its aberrations upon disease. Here we present a computational method for the derivation of a lineage tree from single-cell transcriptome data. By exploiting the tree topology and the transcriptome composition, we establish StemID, an algorithm for identifying stem cells among all detectable cell types within a population. We demonstrate that StemID recovers two known adult stem cell populations, Lgr5+ cells in the small intestine and hematopoietic stem cells in the bone marrow. We apply StemID to predict candidate multipotent cell populations in the human pancreas, a tissue with largely uncharacterized turnover dynamics. We hope that StemID will accelerate the search for novel stem cells by providing concrete markers for biological follow-up and validation.","The dynamics of haematopoietic stem cell differentiation and the hierarchy of oligopotent stem cells in the bone marrow remain controversial. Here we dissect haematopoietic progenitor populations at single cell resolution, deriving an unbiased reference model of transcriptional states in normal and perturbed murine bone marrow. We define the signature of the naive haematopoietic stem cell and find a continuum of core progenitor states. Core cell populations mix transcription of pre-myeloid and pre-lymphoid programs, but do not mix erythroid or megakaryocyte programs with other fates. CRISP-seq perturbation analysis confirms our models and reveals that Cebpa regulates entry into all myeloid fates, while Irf8 and PU.1 deficiency block later differentiation towards monocyte or granulocyte fates. Our transcriptional map defines a reference network model for blood progenitors and their differentiation trajectories during normal and perturbed haematopoiesis.","As core units of organ tissues, cells of various types play their harmonious rhythms to maintain the homeostasis of the human body. It is essential to identify the characteristics of cells in human organs and their regulatory networks for understanding the biological mechanisms related to health and disease. However, a systematic and comprehensive single-cell transcriptional profile across multiple organs of a normal human adult is missing.","Spermatogenesis is a complex and dynamic cellular differentiation process critical to male reproduction and sustained by spermatogonial stem cells (SSCs). Although patterns of gene expression have been described for aggregates of certain spermatogenic cell types, the full continuum of gene expression patterns underlying ongoing spermatogenesis in steady state was previously unclear. Here, we catalog single-cell transcriptomes for >62,000 individual spermatogenic cells from immature (postnatal day 6) and adult male mice and adult men. This allowed us to resolve SSC and progenitor spermatogonia, elucidate the full range of gene expression changes during male meiosis and spermiogenesis, and derive unique gene expression signatures for multiple mouse and human spermatogenic cell types and/or subtypes. These transcriptome datasets provide an information-rich resource for studies of SSCs, male meiosis, testicular cancer, male infertility, or contraceptive development, as well as a gene expression roadmap to be emulated in efforts to achieve spermatogenesis in vitro.","Massively parallel single-cell RNA sequencing can precisely resolve cellular diversity in a high-throughput manner at low cost, but unbiased isolation of intact single cells from complex tissues such as adult mammalian brains is challenging. Here, we integrate sucrose-gradient-assisted purification of nuclei with droplet microfluidics to develop a highly scalable single-nucleus RNA-seq approach (sNucDrop-seq), which is free of enzymatic dissociation and nucleus sorting. By profiling ∼18,000 nuclei isolated from cortical tissues of adult mice, we demonstrate that sNucDrop-seq not only accurately reveals neuronal and non-neuronal subtype composition with high sensitivity but also enables in-depth analysis of transient transcriptional states driven by neuronal activity, at single-cell resolution, in vivo.","Embryonic stem cell (ESC) culture conditions are important for maintaining long-term self-renewal, and they influence cellular pluripotency state. Here, we report single cell RNA-sequencing of mESCs cultured in three different conditions: serum, 2i, and the alternative ground state a2i. We find that the cellular transcriptomes of cells grown in these conditions are distinct, with 2i being the most similar to blastocyst cells and including a subpopulation resembling the two-cell embryo state. Overall levels of intercellular gene expression heterogeneity are comparable across the three conditions. However, this masks variable expression of pluripotency genes in serum cells and homogeneous expression in 2i and a2i cells. Additionally, genes related to the cell cycle are more variably expressed in the 2i and a2i conditions. Mining of our dataset for correlations in gene expression allowed us to identify additional components of the pluripotency network, including Ptma and Zfp640, illustrating its value as a resource for future discovery.","Childhood brain tumors have suspected prenatal origins. To identify vulnerable developmental states, we generated a single-cell transcriptome atlas of >65,000 cells from embryonal pons and forebrain, two major tumor locations. We derived signatures for 191 distinct cell populations and defined the regional cellular diversity and differentiation dynamics. Projection of bulk tumor transcriptomes onto this dataset shows that WNT medulloblastomas match the rhombic lip-derived mossy fiber neuronal lineage and embryonal tumors with multilayered rosettes fully recapitulate a neuronal lineage, while group 2a/b atypical teratoid/rhabdoid tumors may originate outside the neuroectoderm. Importantly, single-cell tumor profiles reveal highly defined cell hierarchies that mirror transcriptional programs of the corresponding normal lineages. Our findings identify impaired differentiation of specific neural progenitors as a common mechanism underlying these pediatric cancers and provide a rational framework for future modeling and therapeutic interventions.","Understanding human embryonic ventral midbrain is of major interest for Parkinson's disease. However, the cell types, their gene expression dynamics, and their relationship to commonly used rodent models remain to be defined. We performed single-cell RNA sequencing to examine ventral midbrain development in human and mouse. We found 25 molecularly defined human cell types, including five subtypes of radial glia-like cells and four progenitors. In the mouse, two mature fetal dopaminergic neuron subtypes diversified into five adult classes during postnatal development. Cell types and gene expression were generally conserved across species, but with clear differences in cell proliferation, developmental timing, and dopaminergic neuron development. Additionally, we developed a method to quantitatively assess the fidelity of dopaminergic neurons derived from human pluripotent stem cells, at a single-cell level. Thus, our study provides insight into the molecular programs controlling human midbrain development and provides a foundation for the development of cell replacement therapies.","Blood glucose levels are tightly controlled by the coordinated action of at least four cell types constituting pancreatic islets. Changes in the proportion and/or function of these cells are associated with genetic and molecular pathophysiology of monogenic, type 1, and type 2 (T2D) diabetes. Cellular heterogeneity impedes precise understanding of the molecular components of each islet cell type that govern islet (dys)function, particularly the less abundant delta and gamma/pancreatic polypeptide (PP) cells. Here, we report single-cell transcriptomes for 638 cells from nondiabetic (ND) and T2D human islet samples. Analyses of ND single-cell transcriptomes identified distinct alpha, beta, delta, and PP/gamma cell-type signatures. Genes linked to rare and common forms of islet dysfunction and diabetes were expressed in the delta and PP/gamma cell types. Moreover, this study revealed that delta cells specifically express receptors that receive and coordinate systemic cues from the leptin, ghrelin, and dopamine signaling pathways implicating them as integrators of central and peripheral metabolic signals into the pancreatic islet. Finally, single-cell transcriptome profiling revealed genes differentially regulated between T2D and ND alpha, beta, and delta cells that were undetectable in paired whole islet analyses. This study thus identifies fundamental cell-type-specific features of pancreatic islet (dys)function and provides a critical resource for comprehensive understanding of islet biology and diabetes pathogenesis.","Multiple myeloma, a plasma cell malignancy, is the second most common blood cancer. Despite extensive research, disease heterogeneity is poorly characterized, hampering efforts for early diagnosis and improved treatments. Here, we apply single cell RNA sequencing to study the heterogeneity of 40 individuals along the multiple myeloma progression spectrum, including 11 healthy controls, demonstrating high interindividual variability that can be explained by expression of known multiple myeloma drivers and additional putative factors. We identify extensive subclonal structures for 10 of 29 individuals with multiple myeloma. In asymptomatic individuals with early disease and in those with minimal residual disease post-treatment, we detect rare tumor plasma cells with molecular characteristics similar to those of active myeloma, with possible implications for personalized therapies. Single cell analysis of rare circulating tumor cells allows for accurate liquid biopsy and detection of malignant plasma cells, which reflect bone marrow disease. Our work establishes single cell RNA sequencing for dissecting blood malignancies and devising detailed molecular characterization of tumor cells in symptomatic and asymptomatic patients.","Oscillatory gene expression is fundamental to development, but technologies for monitoring expression oscillations are limited. We have developed a statistical approach called Oscope to identify and characterize the transcriptional dynamics of oscillating genes in single-cell RNA-seq data from an unsynchronized cell population. Applying Oscope to a number of data sets, we demonstrated its utility and also identified a potential artifact in the Fluidigm C1 platform.","By profiling the transcriptomes of individual cells, single-cell RNA sequencing provides unparalleled resolution to study cellular heterogeneity. However, this comes at the cost of high technical noise, including cell-specific biases in capture efficiency and library generation. One strategy for removing these biases is to add a constant amount of spike-in RNA to each cell and to scale the observed expression values so that the coverage of spike-in transcripts is constant across cells. This approach has previously been criticized as its accuracy depends on the precise addition of spike-in RNA to each sample. Here, we perform mixture experiments using two different sets of spike-in RNA to quantify the variance in the amount of spike-in RNA added to each well in a plate-based protocol. We also obtain an upper bound on the variance due to differences in behavior between the two spike-in sets. We demonstrate that both factors are small contributors to the total technical variance and have only minor effects on downstream analyses, such as detection of highly variable genes and clustering. Our results suggest that scaling normalization using spike-in transcripts is reliable enough for routine use in single-cell RNA sequencing data analyses.","Cells, the basic units of biological structure and function, vary broadly in type and state. Single-cell genomics can characterize cell identity and function, but limitations of ease and scale have prevented its broad application. Here we describe Drop-seq, a strategy for quickly profiling thousands of individual cells by separating them into nanoliter-sized aqueous droplets, associating a different barcode with each cell's RNAs, and sequencing them all together. Drop-seq analyzes mRNA transcripts from thousands of individual cells simultaneously while remembering transcripts' cell of origin. We analyzed transcriptomes from 44,808 mouse retinal cells and identified 39 transcriptionally distinct cell populations, creating a molecular atlas of gene expression for known retinal cell classes and novel candidate cell subtypes. Drop-seq will accelerate biological discovery by enabling routine transcriptional profiling at single-cell resolution. VIDEO ABSTRACT.","T helper 2 (Th2) cells regulate helminth infections, allergic disorders, tumor immunity, and pregnancy by secreting various cytokines. It is likely that there are undiscovered Th2 signaling molecules. Although steroids are known to be immunoregulators, de novo steroid production from immune cells has not been previously characterized. Here, we demonstrate production of the steroid pregnenolone by Th2 cells in vitro and in vivo in a helminth infection model. Single-cell RNA sequencing and quantitative PCR analysis suggest that pregnenolone synthesis in Th2 cells is related to immunosuppression. In support of this, we show that pregnenolone inhibits Th cell proliferation and B cell immunoglobulin class switching. We also show that steroidogenic Th2 cells inhibit Th cell proliferation in a Cyp11a1 enzyme-dependent manner. We propose pregnenolone as a \"lymphosteroid,\" a steroid produced by lymphocytes. We speculate that this de novo steroid production may be an intrinsic phenomenon of Th2-mediated immune responses to actively restore immune homeostasis.","High-throughput single-cell RNA sequencing (scRNA-seq) has become a frequently used tool to assess immune cell heterogeneity. Recently, the combined measurement of RNA and protein expression was developed, commonly known as cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq). Acquisition of protein expression data along with transcriptome data resolves some of the limitations inherent to only assessing transcripts but also nearly doubles the sequencing read depth required per single cell. Furthermore, there is still a paucity of analysis tools to visualize combined transcript-protein datasets. Here, we describe a targeted transcriptomics approach that combines an analysis of over 400 genes with simultaneous measurement of over 40 proteins on 2 × 10","Responses to vaccination and to diseases vary widely across individuals, which may be partly due to baseline immune variations. Identifying such baseline predictors of immune responses and their biological basis is of broad interest, given their potential importance for cancer immunotherapy, disease outcomes, vaccination and infection responses. Here we uncover baseline blood transcriptional signatures predictive of antibody responses to both influenza and yellow fever vaccinations in healthy subjects. These same signatures evaluated at clinical quiescence are correlated with disease activity in patients with systemic lupus erythematosus with plasmablast-associated flares. CITE-seq profiling of 82 surface proteins and transcriptomes of 53,201 single cells from healthy high and low influenza vaccination responders revealed that our signatures reflect the extent of activation in a plasmacytoid dendritic cell-type I IFN-T/B lymphocyte network. Our findings raise the prospect that modulating such immune baseline states may improve vaccine responsiveness and mitigate undesirable autoimmune disease activity.","Oligodendrocytes have been considered as a functionally homogeneous population in the central nervous system (CNS). We performed single-cell RNA sequencing on 5072 cells of the oligodendrocyte lineage from 10 regions of the mouse juvenile and adult CNS. Thirteen distinct populations were identified, 12 of which represent a continuum from Pdgfra(+) oligodendrocyte precursor cells (OPCs) to distinct mature oligodendrocytes. Initial stages of differentiation were similar across the juvenile CNS, whereas subsets of mature oligodendrocytes were enriched in specific regions in the adult brain. Newly formed oligodendrocytes were detected in the adult CNS and were responsive to complex motor learning. A second Pdgfra(+) population, distinct from OPCs, was found along vessels. Our study reveals the dynamics of oligodendrocyte differentiation and maturation, uncoupling them at a transcriptional level and highlighting oligodendrocyte heterogeneity in the CNS.","Conventional human embryonic stem cells are considered to be primed pluripotent but can be induced to enter a naive state. However, the transcriptional features associated with naive and primed pluripotency are still not fully understood. Here we used single-cell RNA sequencing to characterize the differences between these conditions. We observed that both naive and primed populations were mostly homogeneous with no clear lineage-related structure and identified an intermediate subpopulation of naive cells with primed-like expression. We found that the naive-primed pluripotency axis is preserved across species, although the timing of the transition to a primed state is species specific. We also identified markers for distinguishing human naive and primed pluripotency as well as strong co-regulatory relationships between lineage markers and epigenetic regulators that were exclusive to naive cells. Our data provide valuable insights into the transcriptional landscape of human pluripotency at a cellular and genome-wide resolution.","To understand organ function, it is important to have an inventory of its cell types and of their corresponding marker genes. This is a particularly challenging task for human tissues like the pancreas, because reliable markers are limited. Hence, transcriptome-wide studies are typically done on pooled islets of Langerhans, obscuring contributions from rare cell types and of potential subpopulations. To overcome this challenge, we developed an automated platform that uses FACS, robotics, and the CEL-Seq2 protocol to obtain the transcriptomes of thousands of single pancreatic cells from deceased organ donors, allowing in silico purification of all main pancreatic cell types. We identify cell type-specific transcription factors and a subpopulation of REG3A-positive acinar cells. We also show that CD24 and TM4SF4 expression can be used to sort live alpha and beta cells with high purity. This resource will be useful for developing a deeper understanding of pancreatic biology and pathophysiology of diabetes mellitus.","Maintenance of the blood system requires balanced cell fate decisions by hematopoietic stem and progenitor cells (HSPCs). Because cell fate choices are executed at the individual cell level, new single-cell profiling technologies offer exciting possibilities for mapping the dynamic molecular changes underlying HSPC differentiation. Here, we have used single-cell RNA sequencing to profile more than 1600 single HSPCs, and deep sequencing has enabled detection of an average of 6558 protein-coding genes per cell. Index sorting, in combination with broad sorting gates, allowed us to retrospectively assign cells to 12 commonly sorted HSPC phenotypes while also capturing intermediate cells typically excluded by conventional gating. We further show that independently generated single-cell data sets can be projected onto the single-cell resolution expression map to directly compare data from multiple groups and to build and refine new hypotheses. Reconstruction of differentiation trajectories reveals dynamic expression changes associated with early lymphoid, erythroid, and granulocyte-macrophage differentiation. The latter two trajectories were characterized by common upregulation of cell cycle and oxidative phosphorylation transcriptional programs. By using external spike-in controls, we estimate absolute messenger RNA (mRNA) levels per cell, showing for the first time that despite a general reduction in total mRNA, a subset of genes shows higher expression levels in immature stem cells consistent with active maintenance of the stem-cell state. Finally, we report the development of an intuitive Web interface as a new community resource to permit visualization of gene expression in HSPCs at single-cell resolution for any gene of choice.","Systematic analyses of spatiotemporal gene expression trajectories during organogenesis have been challenging because diverse cell types at different stages of maturation and differentiation coexist in the emerging tissues. We identified discrete cell types as well as temporally and spatially restricted trajectories of radial glia maturation and neurogenesis in developing human telencephalon. These lineage-specific trajectories reveal the expression of neurogenic transcription factors in early radial glia and enriched activation of mammalian target of rapamycin signaling in outer radial glia. Across cortical areas, modest transcriptional differences among radial glia cascade into robust typological distinctions among maturing neurons. Together, our results support a mixed model of topographical, typological, and temporal hierarchies governing cell-type diversity in the developing human telencephalon, including distinct excitatory lineages emerging in rostral and caudal cerebral cortex.","Within the bone marrow, stem cells differentiate and give rise to diverse blood cell types and functions. Currently, hematopoietic progenitors are defined using surface markers combined with functional assays that are not directly linked with in vivo differentiation potential or gene regulatory mechanisms. Here, we comprehensively map myeloid progenitor subpopulations by transcriptional sorting of single cells from the bone marrow. We describe multiple progenitor subgroups, showing unexpected transcriptional priming toward seven differentiation fates but no progenitors with a mixed state. Transcriptional differentiation is correlated with combinations of known and previously undefined transcription factors, suggesting that the process is tightly regulated. Histone maps and knockout assays are consistent with early transcriptional priming, while traditional transplantation experiments suggest that in vivo priming may still allow for plasticity given strong perturbations. These data establish a reference model and general framework for studying hematopoiesis at single-cell resolution.","Large-scale surveys of single-cell gene expression have the potential to reveal rare cell populations and lineage relationships but require efficient methods for cell capture and mRNA sequencing. Although cellular barcoding strategies allow parallel sequencing of single cells at ultra-low depths, the limitations of shallow sequencing have not been investigated directly. By capturing 301 single cells from 11 populations using microfluidics and analyzing single-cell transcriptomes across downsampled sequencing depths, we demonstrate that shallow single-cell mRNA sequencing (~50,000 reads per cell) is sufficient for unbiased cell-type classification and biomarker identification. In the developing cortex, we identify diverse cell types, including multiple progenitor and neuronal subtypes, and we identify EGR1 and FOS as previously unreported candidate targets of Notch signaling in human but not mouse radial glia. Our strategy establishes an efficient method for unbiased analysis and comparison of cell populations from heterogeneous tissue by microfluidic single-cell capture and low-coverage sequencing of many cells.","Radial glia, the neural stem cells of the neocortex, are located in two niches: the ventricular zone and outer subventricular zone. Although outer subventricular zone radial glia may generate the majority of human cortical neurons, their molecular features remain elusive. By analyzing gene expression across single cells, we find that outer radial glia preferentially express genes related to extracellular matrix formation, migration, and stemness, including TNC, PTPRZ1, FAM107A, HOPX, and LIFR. Using dynamic imaging, immunostaining, and clonal analysis, we relate these molecular features to distinctive behaviors of outer radial glia, demonstrate the necessity of STAT3 signaling for their cell cycle progression, and establish their extensive proliferative potential. These results suggest that outer radial glia directly support the subventricular niche through local production of growth factors, potentiation of growth factor signals by extracellular matrix proteins, and activation of self-renewal pathways, thereby enabling the developmental and evolutionary expansion of the human neocortex.","How cells respond to myriad stimuli with finite signaling machinery is central to immunology. In naive T cells, the inherent effect of ligand strength on activation pathways and endpoints has remained controversial, confounded by environmental fluctuations and intercellular variability within populations. Here we studied how ligand potency affected the activation of CD8+ T cells in vitro, through the use of genome-wide RNA, multi-dimensional protein and functional measurements in single cells. Our data revealed that strong ligands drove more efficient and uniform activation than did weak ligands, but all activated cells were fully cytolytic. Notably, activation followed the same transcriptional pathways regardless of ligand potency. Thus, stimulation strength did not intrinsically dictate the T cell-activation route or phenotype; instead, it controlled how rapidly and simultaneously the cells initiated activation, allowing limited machinery to elicit wide-ranging responses.","The hypothalamus contains the highest diversity of neurons in the brain. Many of these neurons can co-release neurotransmitters and neuropeptides in a use-dependent manner. Investigators have hitherto relied on candidate protein-based tools to correlate behavioral, endocrine and gender traits with hypothalamic neuron identity. Here we map neuronal identities in the hypothalamus by single-cell RNA sequencing. We distinguished 62 neuronal subtypes producing glutamatergic, dopaminergic or GABAergic markers for synaptic neurotransmission and harboring the ability to engage in task-dependent neurotransmitter switching. We identified dopamine neurons that uniquely coexpress the Onecut3 and Nmur2 genes, and placed these in the periventricular nucleus with many synaptic afferents arising from neuromedin S+ neurons of the suprachiasmatic nucleus. These neuroendocrine dopamine cells may contribute to the dopaminergic inhibition of prolactin secretion diurnally, as their neuromedin S+ inputs originate from neurons expressing Per2 and Per3 and their tyrosine hydroxylase phosphorylation is regulated in a circadian fashion. Overall, our catalog of neuronal subclasses provides new understanding of hypothalamic organization and function.","Hormone-secreting cells within pancreatic islets of Langerhans play important roles in metabolic homeostasis and disease. However, their transcriptional characterization is still incomplete. Here, we sequenced the transcriptomes of thousands of human islet cells from healthy and type 2 diabetic donors. We could define specific genetic programs for each individual endocrine and exocrine cell type, even for rare δ, γ, ε, and stellate cells, and revealed subpopulations of α, β, and acinar cells. Intriguingly, δ cells expressed several important receptors, indicating an unrecognized importance of these cells in integrating paracrine and systemic metabolic signals. Genes previously associated with obesity or diabetes were found to correlate with BMI. Finally, comparing healthy and T2D transcriptomes in a cell-type resolved manner uncovered candidates for future functional studies. Altogether, our analyses demonstrate the utility of the generated single-cell gene expression resource.","Patterns of gene expression can be used to characterize and classify neuronal types. It is challenging, however, to generate taxonomies that fulfill the essential criteria of being comprehensive, harmonizing with conventional classification schemes, and lacking superfluous subdivisions of genuine types. To address these challenges, we used massively parallel single-cell RNA profiling and optimized computational methods on a heterogeneous class of neurons, mouse retinal bipolar cells (BCs). From a population of ∼25,000 BCs, we derived a molecular classification that identified 15 types, including all types observed previously and two novel types, one of which has a non-canonical morphology and position. We validated the classification scheme and identified dozens of novel markers using methods that match molecular expression to cell morphology. This work provides a systematic methodology for achieving comprehensive molecular classification of neurons, identifies novel neuronal types, and uncovers transcriptional differences that distinguish types within a class.","Despite rapid developments in single cell sequencing, sample-specific batch effects, detection of cell multiplets, and experimental costs remain outstanding challenges. Here, we introduce Cell Hashing, where oligo-tagged antibodies against ubiquitously expressed surface proteins uniquely label cells from distinct samples, which can be subsequently pooled. By sequencing these tags alongside the cellular transcriptome, we can assign each cell to its original sample, robustly identify cross-sample multiplets, and \"super-load\" commercial droplet-based systems for significant cost reduction. We validate our approach using a complementary genetic approach and demonstrate how hashing can generalize the benefits of single cell multiplexing to diverse samples and experimental designs.","The primary sensory system requires the integrated function of multiple cell types, although its full complexity remains unclear. We used comprehensive transcriptome analysis of 622 single mouse neurons to classify them in an unbiased manner, independent of any a priori knowledge of sensory subtypes. Our results reveal eleven types: three distinct low-threshold mechanoreceptive neurons, two proprioceptive, and six principal types of thermosensitive, itch sensitive, type C low-threshold mechanosensitive and nociceptive neurons with markedly different molecular and operational properties. Confirming previously anticipated major neuronal types, our results also classify and provide markers for new, functionally distinct subtypes. For example, our results suggest that itching during inflammatory skin diseases such as atopic dermatitis is linked to a distinct itch-generating type. We demonstrate single-cell RNA-seq as an effective strategy for dissecting sensory responsive cells into distinct neuronal types. The resulting catalog illustrates the diversity of sensory types and the cellular complexity underlying somatic sensation.","Nervous systems are composed of various cell types, but the extent of cell type diversity is poorly understood. We constructed a cellular taxonomy of one cortical region, primary visual cortex, in adult mice on the basis of single-cell RNA sequencing. We identified 49 transcriptomic cell types, including 23 GABAergic, 19 glutamatergic and 7 non-neuronal types. We also analyzed cell type-specific mRNA processing and characterized genetic access to these transcriptomic types by many transgenic Cre lines. Finally, we found that some of our transcriptomic cell types displayed specific and differential electrophysiological and axon projection properties, thereby confirming that the single-cell transcriptomic signatures can be associated with specific cellular properties.","Background: A challenge for single-cell genomic studies in kidney and other solid tissues is generating a high-quality single-cell suspension that contains rare or difficult-to-dissociate cell types and is free of both RNA degradation and artifactual transcriptional stress responses.\n\nMethods: We compared single-cell RNA sequencing (scRNA-seq) using the DropSeq platform with single-nucleus RNA sequencing (snRNA-seq) using sNuc-DropSeq, DroNc-seq, and 10X Chromium platforms on adult mouse kidney. We validated snRNA-seq on fibrotic kidney from mice 14 days after unilateral ureteral obstruction (UUO) surgery.\n\nResults: A total of 11,391 transcriptomes were generated in the comparison phase. We identified ten clusters in the scRNA-seq dataset, but glomerular cell types were absent, and one cluster consisted primarily of artifactual dissociation-induced stress response genes. By contrast, snRNA-seq from all three platforms captured a diversity of kidney cell types that were not represented in the scRNA-seq dataset, including glomerular podocytes, mesangial cells, and endothelial cells. No stress response genes were detected. Our snRNA-seq protocol yielded 20-fold more podocytes compared with published scRNA-seq datasets (2.4% versus 0.12%, respectively). Unexpectedly, single-cell and single-nucleus platforms had equivalent gene detection sensitivity. For validation, analysis of frozen day 14 UUO kidney revealed rare juxtaglomerular cells, novel activated proximal tubule and fibroblast cell states, and previously unidentified tubulointerstitial signaling pathways.\n\nConclusions: snRNA-seq achieves comparable gene detection to scRNA-seq in adult kidney, and it also has substantial advantages, including reduced dissociation bias, compatibility with frozen samples, elimination of dissociation-induced transcriptional stress responses, and successful performance on inflamed fibrotic kidney.","Pancreatic islet cells are critical for maintaining normal blood glucose levels, and their malfunction underlies diabetes development and progression. We used single-cell RNA sequencing to determine the transcriptomes of 1,492 human pancreatic α, β, δ, and PP cells from non-diabetic and type 2 diabetes organ donors. We identified cell-type-specific genes and pathways as well as 245 genes with disturbed expression in type 2 diabetes. Importantly, 92% of the genes have not previously been associated with islet cell function or growth. Comparison of gene profiles in mouse and human α and β cells revealed species-specific expression. All data are available for online browsing and download and will hopefully serve as a resource for the islet research community.","The mammalian cerebral cortex supports cognitive functions such as sensorimotor integration, memory, and social behaviors. Normal brain function relies on a diverse set of differentiated cell types, including neurons, glia, and vasculature. Here, we have used large-scale single-cell RNA sequencing (RNA-seq) to classify cells in the mouse somatosensory cortex and hippocampal CA1 region. We found 47 molecularly distinct subclasses, comprising all known major cell types in the cortex. We identified numerous marker genes, which allowed alignment with known cell types, morphology, and location. We found a layer I interneuron expressing Pax6 and a distinct postmitotic oligodendrocyte subclass marked by Itpr2. Across the diversity of cortical cell types, transcription factors formed a complex, layered regulatory code, suggesting a mechanism for the maintenance of adult cell type identity.","The mammalian nervous system executes complex behaviors controlled by specialized, precisely positioned, and interacting cell types. Here, we used RNA sequencing of half a million single cells to create a detailed census of cell types in the mouse nervous system. We mapped cell types spatially and derived a hierarchical, data-driven taxonomy. Neurons were the most diverse and were grouped by developmental anatomical units and by the expression of neurotransmitters and neuropeptides. Neuronal diversity was driven by genes encoding cell identity, synaptic connectivity, neurotransmission, and membrane conductance. We discovered seven distinct, regionally restricted astrocyte types that obeyed developmental boundaries and correlated with the spatial distribution of key glutamate and glycine neurotransmitters. In contrast, oligodendrocytes showed a loss of regional identity followed by a secondary diversification. The resource presented here lays a solid foundation for understanding the molecular architecture of the mammalian nervous system and enables genetic manipulation of specific cell types.","The liver plays a critical role in both immune defense and tolerance in the body. The liver-resident immune cells (LrICs) determine the immune properties, but the unique composition and heterogeneity of these cells are incompletely understood. Here, we dissect the diversity of LrICs by a comprehensive transcriptomic profiling using the unbiased single-cell RNA-sequencing (scRNA-seq). A total of 70, 706 of CD45+ immune cells from the paired liver perfusion, spleen and peripheral blood as references were profiled. We identified more than 30 discrete cell populations comprising 13 of T and NK cell, 7 of B cell, 4 of plasma cell, and 8 of myeloid cell subsets in human liver and donor-paired spleen and blood, and characterized their tissue distribution, gene expression and functional modules. Especially, four of CXCR6+ T and NK cell subsets were found to be present preferentially in the liver, where they manifested heterogeneity, distinct function and prominent homeostatic proliferation. We propose a universal category system of T and NK cells based on distinct chemokine receptors, confirmed subsequently by phenotype, transcriptional factors and functionality. We also identified adaptive changes by the spleen and liver-derived monocyte and macrophage populations. Finally, we give a global glimpse on B cell and plasma cell subsets in human spleen and liver. We, therefore, reveal the heterogeneity and functional diversity of LrICs in human. This study presents comprehensively the landscape of LrICs and will enable further study on their roles in various human diseases.","The mammalian prefrontal cortex comprises a set of highly specialized brain areas containing billions of cells and serves as the centre of the highest-order cognitive functions, such as memory, cognitive ability, decision-making and social behaviour. Although neural circuits are formed in the late stages of human embryonic development and even after birth, diverse classes of functional cells are generated and migrate to the appropriate locations earlier in development. Dysfunction of the prefrontal cortex contributes to cognitive deficits and the majority of neurodevelopmental disorders; there is therefore a need for detailed knowledge of the development of the prefrontal cortex. However, it is still difficult to identify cell types in the developing human prefrontal cortex and to distinguish their developmental features. Here we analyse more than 2,300 single cells in the developing human prefrontal cortex from gestational weeks 8 to 26 using RNA sequencing. We identify 35 subtypes of cells in six main classes and trace the developmental trajectories of these cells. Detailed analysis of neural progenitor cells highlights new marker genes and unique developmental features of intermediate progenitor cells. We also map the timeline of neurogenesis of excitatory neurons in the prefrontal cortex and detect the presence of interneuron progenitors in early developing prefrontal cortex. Moreover, we reveal the intrinsic development-dependent signals that regulate neuron generation and circuit formation using single-cell transcriptomic data analysis. Our screening and characterization approach provides a blueprint for understanding the development of the human prefrontal cortex in the early and mid-gestational stages in order to systematically dissect the cellular basis and molecular regulation of prefrontal cortex function in humans.","Tumor-infiltrating myeloid cells (TIMs) comprise monocytes, macrophages, dendritic cells, and neutrophils, and have emerged as key regulators of cancer growth. These cells can diversify into a spectrum of states, which might promote or limit tumor outgrowth but remain poorly understood. Here, we used single-cell RNA sequencing (scRNA-seq) to map TIMs in non-small-cell lung cancer patients. We uncovered 25 TIM states, most of which were reproducibly found across patients. To facilitate translational research of these populations, we also profiled TIMs in mice. In comparing TIMs across species, we identified a near-complete congruence of population structures among dendritic cells and monocytes; conserved neutrophil subsets; and species differences among macrophages. By contrast, myeloid cell population structures in patients' blood showed limited overlap with those of TIMs. This study determines the lung TIM landscape and sets the stage for future investigations into the potential of TIMs as immunotherapy targets."]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>pmids<\/th>\n      <th>ttxt<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="section level3">
<h3 id="using-it">Using it<a class="anchor" aria-label="anchor" href="#using-it"></a>
</h3>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">scRNAseq</span><span class="op">)</span></span>
<span><span class="va">zld</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/scRNAseq/man/ZilionisLungData.html" class="external-link">ZilionisLungData</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="va">zld</span></span></code></pre></div>
<pre><code><span><span class="co">## class: SingleCellExperiment </span></span>
<span><span class="co">## dim: 41861 173954 </span></span>
<span><span class="co">## metadata(0):</span></span>
<span><span class="co">## assays(1): counts</span></span>
<span><span class="co">## rownames(41861): 5S_rRNA 5_8S_rRNA ... snosnR66 uc_338</span></span>
<span><span class="co">## rowData names(0):</span></span>
<span><span class="co">## colnames(173954): bcIIOD bcHTNA ... bcELDH bcFGGM</span></span>
<span><span class="co">## colData names(16): Library Barcode ... used_in_NSCLC_non_immune</span></span>
<span><span class="co">##   used_in_blood</span></span>
<span><span class="co">## reducedDimNames(5): SPRING_NSCLC_all_cells</span></span>
<span><span class="co">##   SPRING_NSCLC_and_blood_immune SPRING_NSCLC_immune</span></span>
<span><span class="co">##   SPRING_NSCLC_non_immune SPRING_blood</span></span>
<span><span class="co">## mainExpName: NULL</span></span>
<span><span class="co">## altExpNames(0):</span></span></code></pre>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SingleCellExperiment/man/reducedDims.html" class="external-link">reducedDim</a></span><span class="op">(</span><span class="va">zld</span><span class="op">)</span>, pch<span class="op">=</span><span class="st">"."</span>, col<span class="op">=</span><span class="fu"><a href="https://rdrr.io/r/base/factor.html" class="external-link">factor</a></span><span class="op">(</span><span class="va">zld</span><span class="op">$</span><span class="va">`Major cell type`</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<p><img src="A_Data_files/figure-html/getdata-1.png" width="700"></p>
<p>The projection is based on Klein’s <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030950/" class="external-link">SPRING</a>.</p>
<p>Exercises: Supply a legend. Use more modern visualization. Investigate second-level cell type assignments.</p>
</div>
</div>
<div class="section level2">
<h2 id="the-human-cell-atlas">The Human Cell Atlas<a class="anchor" aria-label="anchor" href="#the-human-cell-atlas"></a>
</h2>
<blockquote class="twitter-tweet">
<p lang="en" dir="ltr">
🧬💻🔥 We are excited to launch <a href="https://twitter.com/hashtag/CuratedAtlasQueryR?src=hash&amp;ref_src=twsrc%5Etfw" class="external-link">#CuratedAtlasQueryR</a>! millions of cells at your fingertips. A thread 🧵. <br><br>The harmonized and reannotated <a href="https://twitter.com/hashtag/CELLxGENE?src=hash&amp;ref_src=twsrc%5Etfw" class="external-link">#CELLxGENE</a> <a href="https://twitter.com/hashtag/singlecell?src=hash&amp;ref_src=twsrc%5Etfw" class="external-link">#singlecell</a> human <a href="https://twitter.com/hashtag/atlas?src=hash&amp;ref_src=twsrc%5Etfw" class="external-link">#atlas</a> can be <br>explored and retrieved as <a href="https://twitter.com/hashtag/SingleCellExperiment?src=hash&amp;ref_src=twsrc%5Etfw" class="external-link">#SingleCellExperiment</a>. Please RT!<br><br>-&gt; <a href="https://t.co/8rwqBqPHMQ" class="external-link">https://t.co/8rwqBqPHMQ</a> <a href="https://t.co/2HvHsvmi4H" class="external-link">pic.twitter.com/2HvHsvmi4H</a>
</p>
— Stefano Mangiola (<span class="citation">@steman_research</span>) <a href="https://twitter.com/steman_research/status/1628518254993096705?ref_src=twsrc%5Etfw" class="external-link">February 22, 2023</a>
</blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script><div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/stemangiola/CuratedAtlasQueryR" class="external-link">CuratedAtlasQueryR</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://bioconductor.org/packages/S4Vectors" class="external-link">S4Vectors</a></span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">metadata</span>  <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/CuratedAtlasQueryR/man/get_metadata.html" class="external-link">get_metadata</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="va">metadata</span></span></code></pre></div>
<pre><code><span><span class="co">## <span style="color: #949494;"># Source:   table&lt;/home/stvjc/.cache/R/CuratedAtlasQueryR/metadata.parquet&gt; [?? x 56]</span></span></span>
<span><span class="co">## <span style="color: #949494;"># Database: DuckDB 0.7.1 [stvjc@Linux 5.15.0-67-generic:R 4.3.0/:memory:]</span></span></span>
<span><span class="co">##    .cell          sample_id_db .sample .sample_name assay assay_ontology_term_id</span></span>
<span><span class="co">##    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>          <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>        <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>        <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>                 </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 1</span> AAACCTGAGAGAC… 8a0fe092868… 5f20d7… D17PrPzF_Vi… 10x … EFO:0009899           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 2</span> AAACCTGAGTTGT… 8a0fe092868… 5f20d7… D17PrPzF_Vi… 10x … EFO:0009899           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 3</span> AAACCTGCAGTCG… 02eb2ebcb5f… 5f20d7… D17PrPzF_Vi… 10x … EFO:0009899           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 4</span> AAACCTGCAGTTC… 02eb2ebcb5f… 5f20d7… D17PrPzF_Vi… 10x … EFO:0009899           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 5</span> AAACCTGGTCTAA… 8a0fe092868… 5f20d7… D17PrPzF_Vi… 10x … EFO:0009899           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 6</span> AAACCTGTCGTAC… 8a0fe092868… 5f20d7… D17PrPzF_Vi… 10x … EFO:0009899           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 7</span> AAACCTGTCTTGT… 8a0fe092868… 5f20d7… D17PrPzF_Vi… 10x … EFO:0009899           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 8</span> AAACGGGAGTACG… 8a0fe092868… 5f20d7… D17PrPzF_Vi… 10x … EFO:0009899           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 9</span> AAACGGGAGTAGG… 02eb2ebcb5f… 5f20d7… D17PrPzF_Vi… 10x … EFO:0009899           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;">10</span> AAACGGGAGTGGG… 8a0fe092868… 5f20d7… D17PrPzF_Vi… 10x … EFO:0009899           </span></span>
<span><span class="co">## <span style="color: #949494;"># ℹ more rows</span></span></span>
<span><span class="co">## <span style="color: #949494;"># ℹ 50 more variables: file_id_db &lt;chr&gt;, cell_type &lt;chr&gt;,</span></span></span>
<span><span class="co">## <span style="color: #949494;">#   cell_type_ontology_term_id &lt;chr&gt;, development_stage &lt;chr&gt;,</span></span></span>
<span><span class="co">## <span style="color: #949494;">#   development_stage_ontology_term_id &lt;chr&gt;, disease &lt;chr&gt;,</span></span></span>
<span><span class="co">## <span style="color: #949494;">#   disease_ontology_term_id &lt;chr&gt;, ethnicity &lt;chr&gt;,</span></span></span>
<span><span class="co">## <span style="color: #949494;">#   ethnicity_ontology_term_id &lt;chr&gt;, file_id &lt;chr&gt;, is_primary_data.x &lt;chr&gt;,</span></span></span>
<span><span class="co">## <span style="color: #949494;">#   organism &lt;chr&gt;, organism_ontology_term_id &lt;chr&gt;, …</span></span></span></code></pre>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">metadata</span> <span class="op">|&gt;</span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/count.html" class="external-link">count</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## <span style="color: #949494;"># Source:   SQL [1 x 1]</span></span></span>
<span><span class="co">## <span style="color: #949494;"># Database: DuckDB 0.7.1 [stvjc@Linux 5.15.0-67-generic:R 4.3.0/:memory:]</span></span></span>
<span><span class="co">##          n</span></span>
<span><span class="co">##      <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span></span>
<span><span class="co">## <span style="color: #BCBCBC;">1</span> 28<span style="text-decoration: underline;">975</span>366</span></span></code></pre>
<p>Filtering cells by donor, assay, tissue, cell annotation:</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">single_cell_countsALC</span> <span class="op">=</span> </span>
<span>    <span class="va">metadata</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span></span>
<span>        <span class="va">ethnicity</span> <span class="op">==</span> <span class="st">"African"</span> <span class="op">&amp;</span></span>
<span>        <span class="fu">stringr</span><span class="fu">::</span><span class="fu"><a href="https://stringr.tidyverse.org/reference/str_like.html" class="external-link">str_like</a></span><span class="op">(</span><span class="va">assay</span>, <span class="st">"%10x%"</span><span class="op">)</span> <span class="op">&amp;</span></span>
<span>        <span class="va">tissue</span> <span class="op">==</span> <span class="st">"lung parenchyma"</span> <span class="op">&amp;</span></span>
<span>        <span class="fu">stringr</span><span class="fu">::</span><span class="fu"><a href="https://stringr.tidyverse.org/reference/str_like.html" class="external-link">str_like</a></span><span class="op">(</span><span class="va">cell_type</span>, <span class="st">"%CD4%"</span><span class="op">)</span></span>
<span>    <span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/pkg/CuratedAtlasQueryR/man/get_SingleCellExperiment.html" class="external-link">get_SingleCellExperiment</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## <span style="color: #00BBBB;">ℹ</span> Realising metadata.</span></span></code></pre>
<pre><code><span><span class="co">## <span style="color: #00BBBB;">ℹ</span> Synchronising files</span></span></code></pre>
<pre><code><span><span class="co">## <span style="color: #00BBBB;">ℹ</span> Reading files.</span></span></code></pre>
<pre><code><span><span class="co">## <span style="color: #00BBBB;">ℹ</span> Compiling Single Cell Experiment.</span></span></code></pre>
<div class="sourceCode" id="cb17"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">single_cell_countsALC</span></span></code></pre></div>
<pre><code><span><span class="co">## class: SingleCellExperiment </span></span>
<span><span class="co">## dim: 36229 1571 </span></span>
<span><span class="co">## metadata(0):</span></span>
<span><span class="co">## assays(2): counts cpm</span></span>
<span><span class="co">## rownames(36229): A1BG A1BG-AS1 ... ZZEF1 ZZZ3</span></span>
<span><span class="co">## rowData names(0):</span></span>
<span><span class="co">## colnames(1571): ACAGCCGGTCCGTTAA_F02526_1 GGGAATGAGCCCAGCT_F02526_1 ...</span></span>
<span><span class="co">##   TACAACGTCAGCATTG_SC84_1 CATTCGCTCAATACCG_F02526_1</span></span>
<span><span class="co">## colData names(56): sample_id_db .sample ... n_tissue_in_cell_type</span></span>
<span><span class="co">##   original_cell_id</span></span>
<span><span class="co">## reducedDimNames(0):</span></span>
<span><span class="co">## mainExpName: NULL</span></span>
<span><span class="co">## altExpNames(0):</span></span></code></pre>
<div class="sourceCode" id="cb19"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="fu">colData</span><span class="op">(</span><span class="va">single_cell_countsALC</span><span class="op">)</span><span class="op">[</span>,<span class="fl">1</span><span class="op">:</span><span class="fl">5</span><span class="op">]</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## DataFrame with 6 rows and 5 columns</span></span>
<span><span class="co">##                                     sample_id_db                .sample</span></span>
<span><span class="co">##                                      &lt;character&gt;            &lt;character&gt;</span></span>
<span><span class="co">## ACAGCCGGTCCGTTAA_F02526_1 33cdeb84ae1462d723c1.. 4fc10a6b85e5fa688b25..</span></span>
<span><span class="co">## GGGAATGAGCCCAGCT_F02526_1 33cdeb84ae1462d723c1.. 4fc10a6b85e5fa688b25..</span></span>
<span><span class="co">## TCTTCGGAGTAGCGGT_F02526_1 33cdeb84ae1462d723c1.. 4fc10a6b85e5fa688b25..</span></span>
<span><span class="co">## CCTTACGAGAGCTGCA_F02526_1 33cdeb84ae1462d723c1.. 4fc10a6b85e5fa688b25..</span></span>
<span><span class="co">## ATCTACTCAATGGAAT_F02526_1 33cdeb84ae1462d723c1.. 4fc10a6b85e5fa688b25..</span></span>
<span><span class="co">## CATCAGACACCGTTGG_F00409_1 c7f14e86df84af9ad171.. 9ef5eab00316c489a363..</span></span>
<span><span class="co">##                                     .sample_name       assay</span></span>
<span><span class="co">##                                      &lt;character&gt; &lt;character&gt;</span></span>
<span><span class="co">## ACAGCCGGTCCGTTAA_F02526_1 VUHD92___lung parenc..   10x 5' v1</span></span>
<span><span class="co">## GGGAATGAGCCCAGCT_F02526_1 VUHD92___lung parenc..   10x 5' v1</span></span>
<span><span class="co">## TCTTCGGAGTAGCGGT_F02526_1 VUHD92___lung parenc..   10x 5' v1</span></span>
<span><span class="co">## CCTTACGAGAGCTGCA_F02526_1 VUHD92___lung parenc..   10x 5' v1</span></span>
<span><span class="co">## ATCTACTCAATGGAAT_F02526_1 VUHD92___lung parenc..   10x 5' v1</span></span>
<span><span class="co">## CATCAGACACCGTTGG_F00409_1 VUHD69___lung parenc..   10x 5' v1</span></span>
<span><span class="co">##                           assay_ontology_term_id</span></span>
<span><span class="co">##                                      &lt;character&gt;</span></span>
<span><span class="co">## ACAGCCGGTCCGTTAA_F02526_1            EFO:0011025</span></span>
<span><span class="co">## GGGAATGAGCCCAGCT_F02526_1            EFO:0011025</span></span>
<span><span class="co">## TCTTCGGAGTAGCGGT_F02526_1            EFO:0011025</span></span>
<span><span class="co">## CCTTACGAGAGCTGCA_F02526_1            EFO:0011025</span></span>
<span><span class="co">## ATCTACTCAATGGAAT_F02526_1            EFO:0011025</span></span>
<span><span class="co">## CATCAGACACCGTTGG_F00409_1            EFO:0011025</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="reaching-across-data-available-in-scvi-tools-or-scanpy">Reaching across: data available in scvi-tools or scanpy<a class="anchor" aria-label="anchor" href="#reaching-across-data-available-in-scvi-tools-or-scanpy"></a>
</h2>
<p>Use scviHelper() or scanpyHelper() apps in scviR.</p>
<div class="sourceCode" id="cb21"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/vjcitn/scviR" class="external-link">scviR</a></span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/scviR/man/scviHelper.html" class="external-link">scviHelper</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Vincent Carey.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
